src/main/java/net/coderbot/redwoods/block/BlockConiferSlab.java

```diff
@@ -17,19 +18,28 @@
 public class BlockConiferSlab extends BlockSlab {
 	public static PropertyEnum<EnumSingleVariant> VARIANT = PropertyEnum.create("variant", EnumSingleVariant.class);
 
+	private Block blockDropped;
+
 	public BlockConiferSlab() {
 		super(Material.WOOD);
 
 		setSoundType(SoundType.WOOD);
 		setHardness(2.0F);
 		setResistance(5.0F);
 
+		setBlockDropped(this);
+
 		useNeighborBrightness = true;
 	}
 
+	public void setBlockDropped(Block newBlockDropped)
+	{
+		this.blockDropped = newBlockDropped;
+	}
```

Comment:
Couldn't this just be a constructor argument? The planks are registered before slabs anyways